src/main/fc/runtime_config.h

```diff
@@ -51,10 +51,20 @@ typedef enum {
     ARMING_DISABLED_OSD_MENU        = (1 << 13),
     ARMING_DISABLED_BST             = (1 << 14),
     ARMING_DISABLED_MSP             = (1 << 15),
+#ifdef USE_RUNAWAY_TAKEOFF
```

Comment:
This should not be conditional. If it is, the numbering of flags after it will become target dependent, creating problems for interfacing apps and documentation.

Reply:
Agreed. I think all `USE_RUNAWAY_TAKEOFF` instances should be removed from this header file.